src/silx/gui/widgets/ButtonOverlay.py

```diff
@@ -0,0 +1,11 @@
+from silx.gui import qt
+
+from .OverlayMixIn import _OverlayMixIn
```

Comment:
If the mix-in is private, then it's best if the module is private rather than the class:
```suggestion
from ._OverlayMixIn import OverlayMixIn as _OverlayMixIn
```